TelegramBots/twitterBot/twitterBot.js

```diff
@@ -28,7 +28,11 @@ let params = {screen_name: 'KK_ncnt', since_id: 1, tweet_mode: 'extended'};
 const retrieveLatestTweet = (callback) => {
   client.get('statuses/user_timeline', params, function(error, tweets, response) {
     if (!error) {
-      callback(tweets[0]);
+      if (tweets[0]) {
+        callback(tweets[0]);
+      } else {
+        console.log(tweets);
```

Comment:
I actually thought the tweets object might hold something that is not an empty array. I can't find the case where it would return an empty array since in my testing it always returned at least 1 tweet